src/rules/format.ts

```diff
@@ -79,7 +77,7 @@ const create = (context) => {
   };
 };
 
-export default {
```

Comment:
What was wrong with `export default`?

Reply:
`export default` caused an error, though I can't remember what the message was